<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CombateWidget</class>
 <widget class="QWidget" name="CombateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>851</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="pageCountDown">
    <widget class="CountDownWidget" name="countDownWidget">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageCombat">
    <widget class="QLabel" name="labelPortadaJugador">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>101</width>
       <height>99</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QFrame" name="frameHUDJugador">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>151</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayoutJugador_3"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>149</width>
        <height>65</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QProgressBar" name="barraVidaJug">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="barraKiJug">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="labelNombreJugador">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>197</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelNombreOponente">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>30</y>
       <width>197</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frameHUDOponente">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>60</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>149</width>
        <height>65</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QProgressBar" name="barraVidaOponent">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="barraKiOponent">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="labelPortadaOponente">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>40</y>
       <width>101</width>
       <height>99</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageResultado">
    <widget class="QLabel" name="labelResultado">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>731</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>GANADOR: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRevancha">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>350</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>REVANCHA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVolver">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>350</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>VOLVER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnContinuar">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>350</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>CONTINUAR</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CountDownWidget</class>
   <extends>QLabel</extends>
   <header>CountDownWidget.h</header>
   <slots>
    <signal>finished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
